tst/EnergyPlus/unit/PackagedTerminalHeatPump.unit.cc

```diff
@@ -0,0 +1,509 @@
+		"    ,",
+		"    WSHP;",
+
+		"  Coil:Heating:Electric,",
+		"    Lobby_ZN_1_FLR_2 WSHP Supp Heating Coil,  !- Name",
+		"    OnSched,    !- Availability Schedule Name",
+		"    1.0,                     !- Gas Burner Efficiency",
+		"    Autosize,                   !- Nominal Capacity {W}",
+		"    Lobby_ZN_1_FLR_2 WSHP SuppHeating Coil Air Inlet Node,  !- Air Inlet Node Name",
+		"    Space In Node;       !- Air Outlet Node Name",
+
+		});
+
+		ASSERT_FALSE( process_idf( idf_objects ) );
+
+		bool ErrorsFound( false );
+		GetZoneData( ErrorsFound );
+		GetZoneEquipmentData();
+		GetPTUnit();
+
+		TotNumLoops = 2;
+		PlantLoop.allocate( TotNumLoops );
+
+		for( int l = 1; l <= TotNumLoops; ++l ) {
+			auto & loop( PlantLoop( l ) );
+			loop.LoopSide.allocate( 2 );
+			auto & loopside( PlantLoop( l ).LoopSide( 1 ) );
+			loopside.TotalBranches = 1;
+			loopside.Branch.allocate( 1 );
+			auto & loopsidebranch( PlantLoop( l ).LoopSide( 1 ).Branch( 1 ) );
+			loopsidebranch.TotalComponents = 1;
+			loopsidebranch.Comp.allocate( 1 );
+		}
+		PlantLoop( 2 ).Name = "ChilledWaterLoop";
+		PlantLoop( 2 ).FluidName = "ChilledWater";
+		PlantLoop( 2 ).FluidIndex = 1;
+		PlantLoop( 2 ).FluidName = "WATER";
+		PlantLoop( 2 ).LoopSide( 1 ).Branch( 1 ).Comp( 1 ).Name = VarSpeedCoil( 2 ).Name;
+		PlantLoop( 2 ).LoopSide( 1 ).Branch( 1 ).Comp( 1 ).TypeOf_Num = VarSpeedCoil( 2 ).VSCoilTypeOfNum;
+		PlantLoop( 2 ).LoopSide( 1 ).Branch( 1 ).Comp( 1 ).NodeNumIn = VarSpeedCoil( 2 ).WaterInletNodeNum;
+		PlantLoop( 2 ).LoopSide( 1 ).Branch( 1 ).Comp( 1 ).NodeNumOut = VarSpeedCoil( 2 ).WaterOutletNodeNum;
+
+		PlantLoop( 1 ).Name = "HotWaterLoop";
+		PlantLoop( 1 ).FluidName = "HotWater";
+		PlantLoop( 1 ).FluidIndex = 1;
+		PlantLoop( 1 ).FluidName = "WATER";
+		PlantLoop( 1 ).LoopSide( 1 ).Branch( 1 ).Comp( 1 ).Name = VarSpeedCoil( 1 ).Name;
+		PlantLoop( 1 ).LoopSide( 1 ).Branch( 1 ).Comp( 1 ).TypeOf_Num = VarSpeedCoil( 1 ).VSCoilTypeOfNum;
+		PlantLoop( 1 ).LoopSide( 1 ).Branch( 1 ).Comp( 1 ).NodeNumIn = VarSpeedCoil( 1 ).WaterInletNodeNum;
+		PlantLoop( 1 ).LoopSide( 1 ).Branch( 1 ).Comp( 1 ).NodeNumOut = VarSpeedCoil( 1 ).WaterOutletNodeNum;
+
+		DataSizing::CurZoneEqNum = 1;
+		DataSizing::ZoneSizingRunDone = true;
+		DataSizing::FinalZoneSizing.allocate( 1 );
+		DataSizing::FinalZoneSizing( DataSizing::CurZoneEqNum ).DesCoolVolFlow = 1.0;
+		DataSizing::FinalZoneSizing( DataSizing::CurZoneEqNum ).DesCoolCoilInTemp = 24.0;
+		DataSizing::FinalZoneSizing( DataSizing::CurZoneEqNum ).DesCoolCoilInHumRat = 0.09;
+		DataSizing::FinalZoneSizing( DataSizing::CurZoneEqNum ).CoolDesTemp = 12.0;
+		DataSizing::FinalZoneSizing( DataSizing::CurZoneEqNum ).CoolDesHumRat = 0.05;
+		DataEnvironment::OutBaroPress = 101325;
+		OutputReportPredefined::SetPredefinedTables();
+		DataSizing::ZoneEqSizing.allocate( 1 );
+		DataSizing::ZoneEqSizing( DataSizing::CurZoneEqNum ).SizingMethod.allocate( 16 );
+		SizePTUnit( 1 );
```

Comment:
It should have found the Plant.LoopNum in this unit test.  So it finds the LoopNum if this unit test is run by itself, and doesn't find it if run with all other unit tests? I'll look at this tonight.